binder/plugins/views/file_hash_view.py

```diff
@@ -55,7 +55,7 @@ def _get_params(self, obj, field):
         else:
             return '?' + '&'.join(params)
 
-    def _get_objs(self, queryset, request=None, annotations=None):
+    def _get_objs(self, queryset, *args, **kwargs):
```

Comment:
It may be nice to document this required change to any codebase using this django binder version somewhere? because that would prevent people having to ask/discover they have to perform this change in their codebase in order to use the new binder version. Furthermore this may be a nice moment to start using something like semver in binder?

Reply:
yeah you are  right doing versioning would likely be out of scope for this PR and too much work to implement. lets plan somewhere in the future but this will likely be quite a bit of work.